engine/graphics/src/webgpu/graphics_webgpu.cpp

```diff
@@ -643,120 +698,97 @@ static WGPURenderPipeline WebGPUGetOrCreateRenderPipeline(WebGPUContext* context
         }
         {
             TextureCreationParams params;
-            params.m_Width = width;
-            params.m_Height = height;
+            params.m_Width         = width;
+            params.m_Height        = height;
             params.m_UsageHintBits = 0;
-            textureColor = WebGPUNewTextureInternal(params);
+            textureColor           = WebGPUNewTextureInternal(params);
             WebGPURealizeTexture(textureColor, context->m_Format, 1, context->m_MainRenderTarget->m_Multisample, WGPUTextureUsage_RenderAttachment);
             context->m_MainRenderTarget->m_TextureColor[0] = StoreAssetInContainer(context->m_AssetHandleContainer, textureColor, ASSET_TYPE_TEXTURE);
         }
 
-        WebGPUTexture *textureResolve = GetAssetFromContainer<WebGPUTexture>(context->m_AssetHandleContainer, context->m_MainRenderTarget->m_TextureResolve[0]);
+        WebGPUTexture* textureResolve = GetAssetFromContainer<WebGPUTexture>(context->m_AssetHandleContainer, context->m_MainRenderTarget->m_TextureResolve[0]);
         if (!textureResolve)
         {
-            textureResolve = new WebGPUTexture();
+            textureResolve                                   = new WebGPUTexture();
             context->m_MainRenderTarget->m_TextureResolve[0] = StoreAssetInContainer(context->m_AssetHandleContainer, textureResolve, ASSET_TYPE_TEXTURE);
         }
-        textureResolve->m_Width = width;
+        textureResolve->m_Width  = width;
         textureResolve->m_Height = height;
         textureResolve->m_Format = context->m_Format;
     }
-    //depthstencil
-    WebGPUTexture *textureDepthStencil = GetAssetFromContainer<WebGPUTexture>(context->m_AssetHandleContainer, context->m_MainRenderTarget->m_TextureDepthStencil);
+    // depthstencil
+    WebGPUTexture* textureDepthStencil = GetAssetFromContainer<WebGPUTexture>(context->m_AssetHandleContainer, context->m_MainRenderTarget->m_TextureDepthStencil);
     if (textureDepthStencil)
     {
         delete textureDepthStencil;
         context->m_AssetHandleContainer.Release(context->m_MainRenderTarget->m_TextureDepthStencil);
     }
     {
         TextureCreationParams params;
-        params.m_Width = width;
-        params.m_Height = height;
+        params.m_Width         = width;
+        params.m_Height        = height;
         params.m_UsageHintBits = 0;
-        textureDepthStencil = WebGPUNewTextureInternal(params);
+        textureDepthStencil    = WebGPUNewTextureInternal(params);
         WebGPURealizeTexture(textureDepthStencil, WGPUTextureFormat_Depth24PlusStencil8, 1, context->m_MainRenderTarget->m_Multisample, WGPUTextureUsage_RenderAttachment);
         context->m_MainRenderTarget->m_TextureDepthStencil = StoreAssetInContainer(context->m_AssetHandleContainer, textureDepthStencil, ASSET_TYPE_TEXTURE);
     }
 }
 
-WebGPUContext::WebGPUContext(const ContextParams& params)
```

Comment:
Remove ctor and dctor.

Reply:
Will do as a next piece of changes (I also want to remove `GetMaxElementsIndices` across all graphics adapters)